sdcm/fill_db_data.py

```diff
@@ -285,17 +285,17 @@ class FillDatabaseData(ClusterTester):
             'truncates': ['TRUNCATE limit_ranges_test'],
             'inserts': [
                 "INSERT INTO limit_ranges_test (userid, url, time) VALUES ({}, 'http://foo.{}', 42)".format(_id, tld)
-                for _id in range(0, 100) for tld in ['com', 'org', 'net']],
+                for _id in range(0, 4) for tld in ['com', 'org', 'net']],
             'queries': [
                 "SELECT * FROM limit_ranges_test WHERE token(userid) >= token(2) LIMIT 1",
-                "SELECT * FROM limit_ranges_test WHERE token(userid) > token(2) LIMIT 1"],
+                "SELECT * FROM limit_ranges_test WHERE token(userid) > token(2) LIMIT 1"],  # issue ##2574
```

Comment:
The Scylla issue #2574 had been closed (fixed in https://github.com/scylladb/scylla/issues/2029).

The result only contains one item even without the `limit 1`, there is no order issue of multiple queries.

Reply:
> The Scylla issue #2574 had been closed (fixed in [scylladb/scylla#2029](https://github.com/scylladb/scylla/issues/2029)).

I know. I put this comment just for information
> 
> The result only contains one item even without the `limit 1`, there is no order issue of multiple queries.

Reply:
This select doesn't relate with the scylla issue.